docs/import-export-logic-updates.md

```diff
@@ -0,0 +1,18 @@
+# Import/Export Logic Updates
+
+When making changes to models that are specific to the registry's definition, it's essential to ensure that the import/export logic is updated accordingly.
+This ensures that new fields or models are appropriately extracted and handled during data import/export processes.
+
+## Updating Import/Export Logic
+
+After making relevant changes to the models, it's crucial to update the import/export logic to accommodate these changes. This includes:
+
+1. **Identification of Changes**: Identify the specific changes made to the models and determine how they affect the import/export process.
+
+2. **Modification of Logic**: Modify the import/export logic as necessary to incorporate the new fields or models and handle any changes in data structure or format.
```

Comment:
Can we brainstorm some concrete rules about what to incorporate into the import / export and how?